### DISCUSSION TITLE
How can I find how many pull requests I have created?

### DISCUSSION THREAD

**Discussion by Ice57Block** [community/community#116605] [Pull Requests] [answered]
### Select Topic Area

Product Feedback

### Body

I wanted to check on my progress for the Shark Pull badge and see if I was doing things correctly
Could someone help?
Thanks so much in advance!

**Comment by ethanw2457** [comment, marked as answer]
Hi!
I found that I usually search up
https://github.com/pulls?page=1&q=is%3Apr%20author%3A"USER"%20archived%3Afalse%20is%3Amerged
and replace "USER" with your github username usually works.
Although I do not think this is the most accurate way, it gives a pretty close estimate to getting the total number of Merged PR's
Hope this helps!

**Reply by akostadinov** [reply]
Strange, it shows for me "1057 total" but I don't have the Pull Shark X4 achievement which I see comes at 1024. So probably there is some difference in counting.

**Comment by cosminpm** [comment]
Hello! I think this does not work if you have worked in an organization and you don't have access there anymore 🤔
